Gopkg.lock

```diff
@@ -731,6 +780,59 @@
   pruneopts = "NT"
   revision = "0cf8f7e6ed1d2e3d47d02e3b6e559369af24d803"
 
+[[projects]]
+  digest = "1:9c47d7bd8d8c871437cc55f68096ad6ea37177106351f9d93ef36d59688c9391"
+  name = "k8s.io/kubernetes"
```

Comment:
Wow wow, slow down :D What happened here? Please run `dep ensure` to clean up this and `k8s.io/apiserver` dependencies from the lock file =)

Reply:
Done